diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index 25f06f2..a6e083d 100755
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -73,8 +73,8 @@ else
   # The server platform we are building on.
   KUBE_SERVER_PLATFORMS=(
     linux/amd64
-    linux/arm
-    linux/arm64
+#    linux/arm
+#    linux/arm64
   )
   if [[ "${KUBE_BUILD_PPC64LE:-}" =~ ^[yY]$ ]]; then
     KUBE_SERVER_PLATFORMS+=(linux/ppc64le)
@@ -84,9 +84,9 @@ else
   # The node platforms we build for
   KUBE_NODE_PLATFORMS=(
     linux/amd64
-    linux/arm
-    linux/arm64
-    windows/amd64
+#    linux/arm
+#    linux/arm64
+#    windows/amd64
   )
   if [[ "${KUBE_BUILD_PPC64LE:-}" =~ ^[yY]$ ]]; then
     KUBE_NODE_PLATFORMS+=(linux/ppc64le)
@@ -98,12 +98,12 @@ else
   KUBE_CLIENT_PLATFORMS=(
     linux/amd64
     linux/386
-    linux/arm
-    linux/arm64
-    darwin/amd64
-    darwin/386
-    windows/amd64
-    windows/386
+#    linux/arm
+#    linux/arm64
+#    darwin/amd64
+#    darwin/386
+#    windows/amd64
+#    windows/386
   )
   if [[ "${KUBE_BUILD_PPC64LE:-}" =~ ^[yY]$ ]]; then
     KUBE_CLIENT_PLATFORMS+=(linux/ppc64le)
@@ -113,7 +113,7 @@ else
   # Which platforms we should compile test targets for. Not all client platforms need these tests
   readonly KUBE_TEST_PLATFORMS=(
     linux/amd64
-    darwin/amd64
-    windows/amd64
+#    darwin/amd64
+#    windows/amd64
   )
 fi
